db/schema.rb

```diff
@@ -52,6 +52,8 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.string "profile_photo"
+    t.string "role", default: "member", null: false
```

Comment:
Here I think you've created two different columns for essentially the same thing -- if the system is "user can be an admin or a non-admin", this can be accomplished using just line 56.